invenio_app_ils/ui/backoffice/src/pages/LoanDetails/tests/LoanDetails.test.js

```diff
@@ -1,38 +1,81 @@
 import React from 'react';
-import { shallow, render } from 'enzyme';
+import { mount } from 'enzyme';
+import { Message, Loader } from 'semantic-ui-react';
 import LoanDetails from '../LoanDetails';
+import { LoanMetadata } from '../components/LoanMetadata';
+import { LoanActions } from '../components/LoanActions';
+
+const LOAN_DETAILS_DEFAULT_PROPS = {
+  data: {},
+  error: {},
+  fetchLoading: true,
+  actionLoading: false,
+  loanActionError: false,
+  fetchLoanDetails: () => 'fetchLoanDetails',
+  postLoanAction: () => 'postLoanAction',
+};
 
 describe('tests the LoanDetails component', () => {
-  it('renders the loader component when fetching is in progress', () => {
-    let props = {
-      data: {},
-      error: {},
-      fetchLoading: true,
-      actionLoading: false,
-      loanActionError: false,
-      fetchLoanDetails: () => 'fetchLoanDetails',
-      postLoanAction: () => 'postLoanAction',
+  let component;
+
+  beforeAll(() => {
```

Comment:
Can't you just use `shallow` instead?

Reply:
nope because shallow doesn't render the child components.